custom/icds/icds_toggles.py

```diff
@@ -35,110 +33,95 @@
     'ICDS: Access DISHA API',
     TAG_CUSTOM,
     namespaces=[NAMESPACE_USER],
-    relevant_environments={'icds', 'icds-staging'},
```

Comment:
does this effect the change introduced in the first commit to check for relevant environments?

Reply:
not really since now the toggles will only be registered when the ICDS code is present which will only be on those environments anyway.

Reply:
I meant the changes you made to not check for toggle in https://github.com/dimagi/commcare-hq/pull/28291/commits/a1e5213c797c83f85d102f6ab96f5c70ac6595ba#diff-57f91d9d96727cedf8225cb4c7d50b74L370. Are those changes till relevant now? If an environment will only have toggles for that environment then that improvement isn't really happening?

Reply:
True, but it seems like a reasonable change anyway.